### DISCUSSION TITLE
Taiko marker circle lighting? (osu!lazer)

### DISCUSSION THREAD

**Discussion by KittenzExe** [ppy/osu#23673] [Ideas] [answered]
I was just playing around in osu!lazer Taiko and I noticed that the hit lighting on the marker dosen't match the note that was last played.
(not 100% sure on my terminology but...)
![image](https://github.com/ppy/osu/assets/67358250/b6e000b0-fad9-41c8-b1db-bc7e3e45a7d4)
In the image, there was a blue note played although red lighting.

Is there a possibility to make the hit lighting match the last note played?

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23766
